### ISSUE TITLE
loss value

### ISSUE THREAD

**Issue by vcca** [CasiaFan/tensorflow_retinanet#1]

**Comment by vcca** [comment]
I didn't use the one-hot encoding, but my loss got be NAN.

**Comment by kehaozhe** [comment]
I got the same loss 'NaN'.
At the beginning, I didn't use the one-hot encoding labels, got NaN, After I changed my label to onehot encoding, got NaN too.

**Comment by vcca** [comment]
try to modify the learning rate to a smaller value

**Comment by kehaozhe** [comment]
I got Nan loss in first iteration~
